deploy/helm/sumologic/conf/logs/fluentd/logs.source.default.conf

```diff
@@ -14,14 +14,14 @@
 {{- .Values.fluentd.logs.default.extraFilterPluginConf | nindent 4 }}
 <filter **>
   @type kubernetes_sumologic
-  source_name {{ .Values.fluentd.logs.default.sourceName | quote }}
-  source_category {{ .Values.fluentd.logs.default.sourceCategory | quote }}
-  source_category_prefix {{ .Values.fluentd.logs.default.sourceCategoryPrefix | quote }}
-  source_category_replace_dash {{ .Values.fluentd.logs.default.sourceCategoryReplaceDash | quote }}
-  exclude_facility_regex {{ .Values.fluentd.logs.default.excludeFacilityRegex | quote }}
-  exclude_host_regex {{ .Values.fluentd.logs.default.excludeHostRegex | quote }}
-  exclude_priority_regex {{ .Values.fluentd.logs.default.excludePriorityRegex | quote }}
-  exclude_unit_regex {{ .Values.fluentd.logs.default.excludeUnitRegex | quote }}
+  source_name {{ .Values.sumologic.logs.defaultFluentd.sourceName | quote }}
```

Comment:
Not sure if I understand this change - couldn't we just leave the original properties?

Reply:
The upside of this approach is consistency - everything is mostly in one place, under `sumologic.logs.*`.